queries/locals.scm

```diff
@@ -0,0 +1,79 @@
+; Scopes
+
+[
+  (use_statement) 
+  (actor_definition)
+  (class_definition)
+  (primitive_definition)
+  (interface_definition)
+  (trait_definition)
+  (struct_definition)
+
+  (constructor)
+  (method)
+  (behavior)
+
+  (if_statement)
+  (iftype_statement)
+  (elseif_block)
+  (elseiftype_block)
+  (else_block)
+  (for_statement)
+  (while_statement)
+  (try_statement)
```

Comment:
i think we should add a new scope for the try block, there already is one for the `else` block and one for the `then` block. When I did this query for the former grammar, I though about simply giving every block its own scope. What would you say to that?